### PR TITLE
{lib}[*/2015a] NFFT 3.3.0 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
cc @gppezzi

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/066d30caa685f3131557 for a full test report.

File: easybuild/easyconfigs/n/NFFT/NFFT-3.3.0-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'NFFT'
+version = '3.3.0'
+
+homepage = 'https://www-user.tu-chemnitz.de/~potts/nfft/'
+description = """The NFFT (nonequispaced fast Fourier transform or nonuniform fast Fourier transform) is a C subroutine
+ library for computing the nonequispaced discrete Fourier transform (NDFT) and its generalisations in one or more
+ dimensions, of arbitrary input size, and of complex data."""
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/n/NFFT/NFFT-3.3.0-foss-2015a.eb]: Hi, it looks good overall; 
fyi. I'd capitalize Fast Fourier Transform everywhere, for readability, yet that's up to your liking.

File: easybuild/easyconfigs/n/NFFT/NFFT-3.3.0-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'NFFT'
+version = '3.3.0'
+
+homepage = 'https://www-user.tu-chemnitz.de/~potts/nfft/'
+description = """The NFFT (nonequispaced fast Fourier transform or nonuniform fast Fourier transform) is a C subroutine
+ library for computing the nonequispaced discrete Fourier transform (NDFT) and its generalisations in one or more
+ dimensions, of arbitrary input size, and of complex data."""
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/n/NFFT/NFFT-3.3.0-intel-2015a.eb]: similarly, all good

File: easybuild/easyconfigs/n/NFFT/NFFT-3.3.0-foss-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/n/NFFT/NFFT-3.3.0-foss-2015a.eb]: This is literally copy-pasted from the website, so I'll leave it as is.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 15 out of 15 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6403b1ff30021f188cd8 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @fgeorgatos! Going in...

**Pull request merged**